packages/bull-vault/src/BullStrategy.sol

```diff
@@ -159,4 +170,48 @@ contract BullStrategy is ERC20, LeverageBull {
 
         return (strategyVault.collateralAmount, strategyVault.shortAmount);
     }
+
+    // TODO: Take in crab params when we add full rebalance
+    function _calcDeltaAndCR(bool _isBuyingUsdc, uint256 _usdcAmount) internal view returns (uint256, uint256) {
```

Comment:
I don't think this function should be here, we need to find a way to refactor it and move some of it to LeverageRebalance()

Reply:
I think we should move this to the AuctionBull